index.html

```diff
@@ -1927,79 +908,48 @@ <h4>Error Responses</h4>
       </section>
     </section>
 
-    <!-- External Representation -->
-    <section id="external-representation">
-      <h2 id="external-td-representations">External TD
-        representations</h2>
-      <p>
-        <span class="rfc2119-assertion" id="profile-5.3-exteanal-td-representations-1">
-        The default representation is JSON. Semantic
-        annotations based on JSON-LD MAY be present but are not
-        required to perform all interactions with the thing
-        instance.
-        </span>
+    <!-- Links -->
+    <section id="links">
+      <h3>Links</h3>
+      <p class="ednote">
+        This section will define link relation types which conformant Web Things
+        MAY use and conformant Consumers MAY support
+        (e.g. <code>{ "rel": "alternate", "type": "text/html"}</code> to link
+        to an HTML user interface).
       </p>
+    </section>
 
-        <section id="canonical-representation">
```

Comment:
We need the section on canonical representation, since this may be removed from the TD.

Reply:
What is the canonical representation used for in the WoT Profile specification, and why do you think it may be removed from the Thing Description specification?